packages/react-card/src/components/Card/Card.types.ts

```diff
@@ -6,6 +6,7 @@ export type CardSlots = {
 
 export type CardCommons = {
   appearance: 'filled' | 'filled-alternative' | 'outline' | 'subtle';
+  focusable: boolean | 'noTab' | 'tabExit' | 'tabOnly';
```

Comment:
This is really obvious because they are right next to each other but L9 we have camel case string values and L8 we have kebab case. For consistency we should pick one and stick with it.